src/Nethermind/Nethermind.Network.Discovery/Discv5/DiscoveryV5App.cs

```diff
@@ -90,18 +92,20 @@ .. _discoveryDb.GetAllValues().Select(enr => enrFactory.CreateFromBytes(enr, ide
             .WithEntry(EnrEntryKey.Tcp, new EntryTcp(_networkConfig.P2PPort))
             .WithEntry(EnrEntryKey.Udp, new EntryUdp(_networkConfig.DiscoveryPort));
 
-        _discv5Protocol = new Discv5ProtocolBuilder(services)
+        IDiscv5ProtocolBuilder discv5Builder = new Discv5ProtocolBuilder(services)
             .WithConnectionOptions(new ConnectionOptions
             {
                 UdpPort = _networkConfig.DiscoveryPort,
             })
             .WithSessionOptions(sessionOptions)
             .WithTableOptions(new TableOptions(bootstrapEnrs.Select(enr => enr.ToString()).ToArray()))
             .WithEnrBuilder(enrBuilder)
-            .WithLoggerFactory(new NethermindLoggerFactory(logManager, true))
-            .Build();
-
+            .WithLoggerFactory(new NethermindLoggerFactory(logManager, true));
 
+        discv5Builder.Build(); // Force-add default services
```

Comment:
This flow can be simplified once Lantern.Discv5 allows to override internal services.
I'll add a simple PR to their repository, once this implementation is agreed upon.